### PR TITLE
MSDKUI-1635: [UI Kit - Android] Build app to demonstrate raw components

### PR DESCRIPTION
**Pull request by imvjk**
Added initial code

Signed-off-by: Vijay Yadav <5671997+imvjk@users.noreply.github.com>

### CODE REVIEW

File: MSDKUIDev/app/proguard-rules.pro [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```
Comment (stefan-troyke) [inline MSDKUIDev/app/proguard-rules.pro]: Let's remove the file. We don't need it.

File: MSDKUIDev/app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        minSdkVersion 19
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
```
Comment (stefan-troyke) [inline MSDKUIDev/app/build.gradle]: let's remove proguard, do we need it?

Comment (imvjk) [reply MSDKUIDev/app/build.gradle]: I think we should keep it. getDefaultProguardFile('proguard-android.txt') method gets the default ProGuard settings file which is used to shrink code removes unused classes, fields, methods, and attributes so overall making apk optmized and small. 

proguard-rules.pro is for project specific setting which of course we don't need it now but i don't see any harm to keep it here for future perspective.

File: MSDKUIDev/app/src/main/java/com/here/msdkuidev/ComponentSettingList.kt [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        settingMap = intent.getSerializableExtra(COMPONENT) as Setting<*>
+        setUpList(settingMap.getItems(this@ComponentSettingList).keys.map { key-> Pair(key, "") }, itemClickListener)
+    }
+
+    private val itemClickListener = object : LandingScreenAdapter.Listener {
+        override fun onItemClicked(view: View) {
+            val position = landing_list.getChildLayoutPosition(view)
+            val item = settingMap.getItems(this@ComponentSettingList).values.toList()[position]
+            item.subTitle = settingMap.subTitle
+           startActivity(Intent(this@ComponentSettingList, settingMap.getClassName()).apply {
```
Comment (tstaszak89) [inline MSDKUIDev/app/src/main/java/com/here/msdkuidev/ComponentSettingList.kt]: Incorrect indentation (3 spaces instead of 4).

**Pull request merged**